File: csharp/src/main/arrays/StaticArray.cs
using System;

namespace arrays
{
    public class StaticArray
    {
        int[] numbers;

        public StaticArray(int[] numbers)
        {
            this.numbers = numbers;
        }

        public bool find(int num)
        {
            bool found = false;
            for (int i = 0; i < numbers.Length; i++)
            {
                if (numbers[i] == num)
                {
                    found = true;
                    break;
                }
            }

            return found;
        }

        internal void printElementsWithIndexes()
        {
            Console.WriteLine($"Array Size = {numbers.Length}");
            for (int i = 0; i < numbers.Length; i++)
            {
                Console.WriteLine($"numbers[{i}] = {numbers[i]}");
            }

        }
    }
}

File: java/src/main/java/arrays/StaticArray.java
package arrays;

/**
 * StaticArray
 */
public class StaticArray {
    int[] numbers;

    public StaticArray(int[] numbers) {
        this.numbers = numbers;
    }

    public boolean find(int num) {
        boolean found = false;
        for (int i = 0; i < numbers.length; i++) {
            if (numbers[i] == num) {
                found = true;
                break;
            }
        }

        return found;
    }

    public void printElementsWithIndexes() {
        System.out.println(String.format("Array Size = %s", numbers.length));
        for (int i = 0; i < numbers.length; i++) {
            System.out.println(String.format("numbers[%s]  = %s", i, numbers[i]));
        }
    }
}